File: app/src/main/java/com/lifedawn/bestweather/ui/findaddress/map/IBottomSheetState.java
package com.lifedawn.bestweather.ui.findaddress.map;

public interface IBottomSheetState {
	void setStateOfBottomSheet(BottomSheetType bottomSheetType, int state);

	int getStateOfBottomSheet(BottomSheetType bottomSheetType);

	void collapseAllExpandedBottomSheets();
}

File: app/src/main/java/com/lifedawn/bestweather/ui/findaddress/map/IBottomSheetState.kt
package com.lifedawn.bestweather.ui.findaddress.map

interface IBottomSheetState {
    fun setStateOfBottomSheet(bottomSheetType: BottomSheetType, state: Int)
    fun getStateOfBottomSheet(bottomSheetType: BottomSheetType): Int
    fun collapseAllExpandedBottomSheets()
}